src/gui/FileBrowser.cpp

```diff
@@ -262,7 +263,6 @@ void FileBrowser::giveFocusToFilter()
 }
 
 
-
```

Comment:
This whitespace seems superfluous. We wouldn't normally fault for this but there's a second instance now introduced from a reverted commit, might as well fix them both.

Reply:
alright, btw. are we planing to set up an auto formater (clang-format for instance) or is there another way to enforce the code styles? In the Wiki I could only find which rules to follow.

Reply:
Yes, we'll be doing just that (clang-tidy) in the future.  Thanks for the concerns, consideration and patience.  The codebase is undergoing a lot of refactoring which will greatly reduced this type of back-and-forth.